src/stories/components/Slider/Slider.js

```diff
@@ -0,0 +1,91 @@
+// Awesome Swiper React component credits: https://swiperjs.com/react
+
+import './styles.scss';
+import 'swiper/swiper.scss'; // Swiper styles
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Swiper, SwiperSlide } from 'swiper/react';
+
+const TABLET_SCREEN_SIZE = 640;
+const DESKTOP_SCREEN_SIZE = 1024;
+const WIDESCREEN_SCREEN_SIZE = 1400;
+
+
+/**
+ * Component for branding element.
+ *
+ * @component
+ * @param {node} children The items that will be shown as a list in the slider.
+ * @return {object} (
+ *   <Branding modifierClasses={modifierClasses}
+ *             text={text}
+ *   />
+ * )
+ */
+export const Slider = ({ children }) => {
+  // Swiper reference to control scrolling via Buttons
+  const [swiper, setSwiper] = useState(null);
+
+  // Put each child of the component into a separate slide
+  const slides = children.map((child, i) =>
+    <SwiperSlide key={i}>
+      {child}
+    </SwiperSlide>);
+
+  return (
+    <div>
+      {/* Slider itself */}
+      <Swiper
+        spaceBetween={30}
+        freeMode={true}
+        onSwiper={(swiperInstance) => {
+          // Save this Swiper's instance when initialized
+          setSwiper(swiperInstance);
+        }}
+        breakpoints={{
+          // tablet
+          640: {
+            width: TABLET_SCREEN_SIZE,
+            slidesPerView: 2,
+          },
+          // desktop
+          1024: {
+            width: DESKTOP_SCREEN_SIZE,
+            slidesPerView: 3,
+          },
+          // widescreen
+          1400: {
+            width: WIDESCREEN_SCREEN_SIZE,
+            slidesPerView: 4,
+          },
+        }}
+      >
+        {slides}
+      </Swiper>
+
+      {/* Slider control buttons */}
+      <div className="slider__controls">
+        <button
+          className='slider__button slider__button--back'
+          onClick={() => swiper.slidePrev()}
+        >
+          BACK
+        </button>
+        <button
+          className='slider__button slider__button--next'
```

Comment:
When you reach the end or the start of the slider the arrows stop working; we should include a "disabled" style to alert users the arrows are no longer functional. Or we can enable an infinite loop.